client/src/components/relatedItems/RelatedItemsWidget.jsx

```diff
@@ -94,11 +190,13 @@ class RelatedItemsWidget extends Component {
 
     return (
       <>
-        <div className="related-items-widget">
-          <RelatedProducts overviewId={this.state.overviewId} overviewIdName={this.state.overviewIdName} overviewIdFeatures={this.state.overviewIdFeatures} relatedProductsIds={this.state.relatedProductsIds} setOverviewId={this.setOverviewId} />
-          <WrappedYourOutfit />
-          <div id="comparison-modal-overlay"></div>
-        </div>
+        {this.state.loaded === true &&
```

Comment:
'this.state.loaded === true' can be shortened to 'this.state.loaded'

Reply:
Ooh good catch!  Thank you!